### PR TITLE
fixes #19453 - applies postgres group owner to whole backup path

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

File: katello/katello-backup [hunk 1]
```diff
@@ -54,6 +54,17 @@ def run_cmd(command, exit_codes=[0])
   end
 end
 
+def create_directories(directory)
+  @dir << "/" unless @dir.end_with? "/"
+  @dir << "katello-backup-" + DateTime.now.strftime('%Y%m%d%H%M%S')
+  puts "Starting backup: #{Time.now}"
```
Comment (evgeni) [inline katello/katello-backup]: I think the "Starting backup" message should live outside of `create_directories`, as the fact that the backup is starting has nothing to do with the creation of the folder.

File: katello/katello-backup [hunk 2]
```diff
@@ -54,6 +54,17 @@ def run_cmd(command, exit_codes=[0])
   end
 end
 
+def create_directories(directory)
+  @dir << "/" unless @dir.end_with? "/"
+  @dir << "katello-backup-" + DateTime.now.strftime('%Y%m%d%H%M%S')
```
Comment (evgeni) [inline katello/katello-backup]: Not really related to this PR, but is there any reason not to use
`@dir = File.join(directory, "katello-backup-" + DateTime.now.strftime('%Y%m%d%H%M%S'))`
here?

**Comment by evgeni** [comment]
besides the two non-critical comments LGTM!

**Review by evgeni — APPROVED** [review]

File: katello/katello-backup (hunk 1 above)
Comment (cfouant) [reply katello/katello-backup]: I agree, I was on the fence with that one. Making changes now :+1:

**Comment by evgeni** [comment]
ACK

**Comment by cfouant** [comment]
@ehelms @jlsherrill can I have merge pls?

**Comment by jlsherrill** [comment]
Merged, thanks @cfouant !

**Pull request merged**